### PR TITLE
Fixes #32 Improved question representation

### LINKED ISSUE #32: Improve initial question representation
For now it shows: 
capture 
It looks very clumsy and odd. Try to improve it and make it more specific and clear.

### PR DESCRIPTION
**Pull request by thesmallstar**
I am attaching appropriate SS. 
Changes: 
1. All single quotes were replaced by double quotes by my VSC prettier I am not very sure why. 
2. I have merged with upstream/master Still I see 2 unchanged files contributing.md and package-lock.json Not sure for that as well. :) 

fcli

### CODE REVIEW

**Comment by debck** [comment]
The error log says this: 

capture

So try to run `npm install ` again and then `npm run test` . And commit only when all the test passes. Let me know where you get stuck. 😄

**Comment by thesmallstar** [comment]
@debck  I am not sure about contributing.md, any idea?

File: bin/sub.js [hunk 1]
```diff
@@ -40,7 +42,11 @@ exports.main = (exports.main = async() => {
   } else if (isdir === 'rf') {
     func.renamefile();
   } else {
-    // Invalid response in isdir
-    return 2;
+    console.log(
+      chalk.green(
+        'Commands for-\n 1. Directory\n Create: d\n Delete: dld\n' +
```
Comment (debck) [inline bin/sub.js]: Don't write it under `else` section. It should be inside another `else if` .

File: bin/sub.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ const ifdir = require('../lib/ifdir');
 
 const func = require('../lib/functions');
 
-exports.main = (exports.main = async() => {
+exports.main = exports.main = async() => {
```
Comment (debck) [inline bin/sub.js]: `exports.main ` is  duplicated here.

File: bin/sub.js (hunk 1 above)
Comment (debck) [reply bin/sub.js]: use some other command like else if ( isdir === 'h' || isdir === 'help'). Something like this

**Comment by thesmallstar** [comment]
@debck I kept them in else condition so that if the user enters the wrong command he is directly told what the correct commands are. I will add the check for 'h' or 'help' now

**Comment by debck** [comment]
@thesmallstar  Add for both using `OR` .

**Pull request merged**